File: 프로그래머스/0/120848. 팩토리얼/팩토리얼.java
class Solution {
    public int solution(int n) {
        int box=0;
        int answer = 0;
        int sum=1;
        for(int i=1;i<=n;i++){
            sum*=i;
            if(sum>n){
                return i-1;
            }
            else if(sum==n){
                return i;
            }
           
            }
        return 0;
    }
}

File: 프로그래머스/0/120848. 팩토리얼/팩토리얼.js
function solution(n) {
    let sum=1;
    for(let i=1;i<=n;i++){
        sum=1;
        for(let j=1;j<=i;j++){
            sum*=j;
            if (sum==n){
                return i;
            }
            if(sum>n){
                return i-1;
            }
        }
    }
}